src/App.scss

```diff
@@ -73,84 +89,135 @@ body {
     margin-left: 50px;
 }
 
-.App__program-section {
-    margin-top: 1.5rem;
+.App__world-selector-container {
+    grid-column-start: 4;
+    grid-column-end: 5;
+    grid-row-start: 1;
+    grid-row-end: 3;
+    margin-right: 0.5rem;
+    width: 6rem;
```

Comment:
Let's apply the same amount of margin as the scene to the top. 

`
margin-top: 0.5rem;
`

Reply:
Done.